docs/sources/administration/configuration.md

```diff
@@ -1909,3 +1909,9 @@ default_baselayer_config = `{
 ### enable_custom_baselayers
 
 Set this to `true` to disable loading other custom base maps and hide them in the Grafana UI. Default is `false`.
+
+## [dashboard_previews]
+
+> **Note:** This feature is available in Grafana v9.0 and later versions.
```

Comment:
Move the note to after line 1917. Also, I do not see the configuration option for this feature. Should we add it?

Reply:
configuration moved out of the main file